### PR TITLE
Implement document deletion

### PR DESCRIPTION
**Pull request by ethankent**
Related to this issue - https://github.com/pinax/pinax-documents/issues/3

### CODE REVIEW

**Comment by ethankent** [comment]
I didn't realize I'd need to do testing for coveralls. I'll take a look at that and update.

File: pinax/documents/views.py [hunk 1]
```diff
@@ -273,6 +273,9 @@ class DocumentDelete(LoginRequiredMixin, DeleteView):
     model = Document
     success_url = reverse_lazy("pinax_documents:document_index")
 
+    def get_template_names(self):
```
Comment (paltman) [inline pinax/documents/views.py]: @ethankent  Can we just specify the the template_name on the class directly?

**Comment by paltman** [comment]
@ethankent looks great. Be merged your PR on the theme and left a comment about one small thing I'd like to see fixed.

File: pinax/documents/views.py (hunk 1 above)
Comment (ethankent) [inline pinax/documents/views.py]: Hi @paltman 

Thanks for the update. Just wanted to clarify your question a bit. I think you're asking if the template name can be specified in the same way that template_name is specified in the DocumentDetail class. The answer is yes!

For some reason I was thinking that the template name _needed_ to be calculated dynamically (for instance showing a different template for post() than for get()). I suppose if we wanted to show a confirmation message that might be a good way to go. But we don't, we want to redirect upon successful deletion. So, specifying the static class variable template_name seems like a no-brainer now. Thanks for the feedback on that. I'll update right away.

**Pull request merged**